### PR TITLE
area_spooky

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀 This PR matches 2 functions (+0.00%) on `us`.

File: src/world/area_obk/obk_05/obk_05_2_main.c [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        return ApiStatus_BLOCK;
+    }
+}
+
+EvtScript N(EVS_ManageRetroJar) = {
+    EVT_LOOP(0)
+        EVT_CALL(N(RetroJar_AwaitPlayerEntry))
+        EVT_CALL(DisablePlayerInput, TRUE)
+        EVT_WAIT(1)
+        EVT_IF_EQ(MF_Unk_0A, FALSE)
```
Comment (nanaian) [inline src/world/area_obk/obk_05/obk_05_2_main.c:63]: Could you alias the mapflag to MF_Is8bit or similar?

File: src/world/area_kmr/kmr_22/kmr_22_2_npc.c [hunk 2]
```diff
@@ -0,0 +1,809 @@
...
+#include "npc/chapter_1.inc.c"
+#include "npc/chapter_2.inc.c"
+#include "npc/chapter_3.inc.c"
+#include "npc/chapter_4.inc.c"
+#include "npc/chapter_5.inc.c"
+#include "npc/chapter_6.inc.c"
+#include "npc/chapter_7.inc.c"
+#include "npc/chapter_8.inc.c"
+
+StaticNpc N(NpcData_Chapter1)[] = {
```
Comment (nanaian) [inline src/world/area_kmr/kmr_22/kmr_22_2_npc.c:33]: It's a shame that the npc lists come after the block of includes

Comment (HailSanta) [reply src/world/area_kmr/kmr_22/kmr_22_2_npc.c:33]: Its pretty standard in the map data for the NpcData structs to all come together after the npc scripts. Additionally, the NpcSettings structs are usually found first (before the scripts) as part of some set of includes.

**Pull request merged**